flixel/system/frontEnds/CameraFrontEnd.hx

```diff
@@ -14,6 +14,12 @@ class CameraFrontEnd
 	 * By default flixel creates one camera the size of the screen.
 	 */
 	public var list(default, null):Array<FlxCamera> = [];
```

Comment:
Note, I ended up keeping this public, [FlxTilemap uses it](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/tile/FlxTilemap.hx#L385) and it may be handy to have a list to iterate. I considered copying haxe 4's [ReadOnlyArray](https://github.com/HaxeFoundation/haxe/blob/development/std/haxe/ds/ReadOnlyArray.hx) but I'm not sure if that class would work on 3.4.7